webpack/JobInvocationDetail/JobInvocationHostTable.js

```diff
@@ -26,52 +27,71 @@ import PropTypes from 'prop-types';
 import React, { useEffect, useMemo, useState } from 'react';
 import { FormattedMessage } from 'react-intl';
 import { useHistory } from 'react-router-dom';
-import URI from 'urijs';
 import { CheckboxesActions } from './CheckboxesActions';
 import DropdownFilter from './DropdownFilter';
 import Columns, {
   JOB_INVOCATION_HOSTS,
-  MAX_HOSTS_API_SIZE,
-  STATUS_UPPERCASE,
   LIST_TEMPLATE_INVOCATIONS,
+  STATUS_UPPERCASE,
   ALL_JOB_HOSTS,
 } from './JobInvocationConstants';
-import { PopupAlert } from './OpenAllInvocationsModal';
 import { TemplateInvocation } from './TemplateInvocation';
 import { RowActions } from './TemplateInvocationComponents/TemplateActionButtons';
 
 const JobInvocationHostTable = ({
-  failedCount,
   id,
   initialFilter,
   onFilterUpdate,
   targeting,
 }) => {
   const columns = Columns();
   const columnNamesKeys = Object.keys(columns);
-  const apiOptions = { key: JOB_INVOCATION_HOSTS };
+
   const history = useHistory();
-  const [selectedFilter, setSelectedFilter] = useState(initialFilter);
-  const [expandedHost, setExpandedHost] = useState([]);
+  const dispatch = useDispatch();
 
-  useEffect(() => {
-    if (initialFilter !== selectedFilter) {
-      wrapSetSelectedFilter(initialFilter);
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [initialFilter]);
+  const [apiResponse, setApiResponse] = useState([]);
+  const [status, setStatus] = useState(STATUS_UPPERCASE.PENDING);
+
+  const [allHostsIds, setAllHostsIds] = useState([]);
 
+  // Expansive items
+  const [expandedHost, setExpandedHost] = useState([]);
+  const isHostExpanded = host => expandedHost.includes(host);
+  const setHostExpanded = (host, isExpanding = true) =>
+    setExpandedHost(prevExpanded => {
+      const otherExpandedHosts = prevExpanded.filter(h => h !== host);
+      return isExpanding ? [...otherExpandedHosts, host] : otherExpandedHosts;
+    });
+
+  // Page table params
+  // Parse URL
   const {
     searchParam: urlSearchQuery = '',
     page: urlPage,
     per_page: urlPerPage,
     order: urlOrder,
   } = useUrlParams();
 
-  const constructFilter = (
-    filter = selectedFilter,
-    search = urlSearchQuery
-  ) => {
+  // default
+  const defaultParams = useMemo(
+    () => ({
+      page: urlPage ? Number(urlPage) : 1,
+      ...(urlPerPage && { per_page: Number(urlPerPage) }),
+      order: urlOrder || '',
```

Comment:
If I change the order to name desc, and the change the page the order gets ignored. Also it doesnt seem to go to the url which might be the reason